packages/github/README.md

```diff
@@ -55,3 +55,19 @@ const newIssue = await octokit.issues.create({
   body: 'Hello Universe!'
 });
 ```
+
+## Webhook Payload Typescript Definitions
```

Comment:
Consider sentence-case rather than title-case. Sentence-case seems to be the common trend, e.g. [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-github-actions)